pkg/convert/testdata/programs/sandbox_registry_module/pcl/main.pp

```diff
@@ -0,0 +1,24 @@
+package "subnets" {
+  baseProviderName    = "terraform-module"
+  baseProviderVersion = "0.1.3"
+  parameterization {
+    name    = "subnets"
+    version = "1.0.0"
```

Comment:
This is preserved from main.tf `version = "1.0.0"` right?

Reply:
Yes